.devcontainer/mysql/devcontainer.json

```diff
@@ -1,10 +1,10 @@
 {
     "name": "Apache Airflow - mysql",
     "dockerComposeFile": [
-        "../scripts/ci/docker-compose/devcontainer.yml",
-        "../scripts/ci/docker-compose/local.yml",
-        "../scripts/ci/docker-compose/backend-mysql.yml",
-        "../scripts/ci/docker-compose/devcontainer-mysql.yml"
+        "../../scripts/ci/docker-compose/devcontainer.yml",
```

Comment:
Ah cool. I am not sure why it does not work but maybe the early version of "muliple codespaces versions" implementation had a different workdir setting :P. Thanks for that one :)